app/view_models/prior_authority/service_costs_with_adjustments.rb

```diff
@@ -0,0 +1,138 @@
+      original_items
+    end
+
+    def requested_period
+      original_period
+    end
+
+    def requested_base_cost
+      if cost_type == 'per_item'
+        requested_items * requested_cost_per_item
+      else
+        (
+          (requested_period.hours * requested_cost_per_hour) +
+          ((requested_period.minutes / 60.0) * requested_cost_per_hour)
+        ).round(2)
+      end
+    end
+
+    def requested_base_cost_per_unit
+      NumberTo.pounds(requested_cost_per_unit)
+    end
+
+    def requested_cost_per_unit
+      @requested_cost_per_unit ||= cost_type == 'per_item' ? requested_cost_per_item : requested_cost_per_hour
+    end
+
+    def requested_cost_per_item
+      original_cost_per_item
+    end
+
+    def requested_cost_per_hour
+      original_cost_per_hour
+    end
+  end
+
+  module AdjustedServiceCosts
+    include RequestedServiceCosts
+
+    def adjusted_humanized_units
+      return unless adjusted_items || adjusted_period
+
+      if cost_type == 'per_item'
+        "#{adjusted_items} " \
+          "#{I18n.t("prior_authority.application_details.items.#{item_type}").pluralize(adjusted_items)}"
+      else
+        format_period(adjusted_period)
+      end
+    end
+
+    def adjusted_humanized_cost_per_unit
+      return unless adjusted_formatted_cost_per_unit
+
+      i18n_key = cost_type == 'per_item' ? "per_#{item_type}" : cost_type
+
+      "#{adjusted_formatted_cost_per_unit} " \
+        "#{I18n.t(i18n_key, scope: 'prior_authority.application_details.items.per_unit_descriptions')}"
+    end
+
+    def adjusted_formatted_service_cost_total
+      NumberTo.pounds(adjusted_service_cost_total) if adjusted_service_cost_total
+    end
+
+    private
+
+    def adjusted_formatted_cost_per_unit
+      NumberTo.pounds(adjusted_cost_per_unit) if adjusted_cost_per_unit
+    end
+
+    def adjusted_cost_per_unit
+      @adjusted_cost_per_unit ||= cost_type == 'per_item' ? adjusted_cost_per_item : adjusted_cost_per_hour
+    end
+
+    def adjusted_service_cost_total
+      cost_type == 'per_item' ? adjusted_item_cost_total : adjusted_hour_cost_total
+    end
+
+    def adjusted_item_cost_total
+      return unless adjusted_items && adjusted_cost_per_item
+
+      adjusted_items * adjusted_cost_per_item
+    end
+
+    def adjusted_hour_cost_total
+      return unless adjusted_period && adjusted_cost_per_hour
+
+      (
+        (adjusted_period.hours * adjusted_cost_per_hour) +
+        ((adjusted_period.minutes / 60.0) * adjusted_cost_per_hour)
+      ).round(2)
+    end
+
+    def adjusted_items
+      items if items_original || cost_per_item_original
+    end
+
+    def adjusted_period
+      period if period_original || cost_per_hour_original
+    end
+
+    def adjusted_cost_per_item
+      cost_per_item if cost_per_item_original || items_original
+    end
+
+    def adjusted_cost_per_hour
+      cost_per_hour if cost_per_hour_original || period_original
+    end
+  end
+
+  module ServiceCostsWithAdjustments
```

Comment:
not sure I understand the benefit of splitting this into multiple modules?

Reply:
this came out of having one monster of a module, originally, violating rubocop module metrics, and me iterating on breaking it down. I could separate the modules, for this and travel costs, making 4 modules, and add them to the quote class individually or leave like this. what you think?

Reply:
This is fine... might be worth adding as a comment so people understand why it's like this... :)